inference-engine/ie_bridges/python/tests/test_ExecutableNetwork.py

```diff
@@ -309,6 +309,16 @@ def test_get_config(device):
     assert config == "NO"
 
 
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "GNA", reason="Device dependent test")
+def test_set_config(device):
+    ie_core = ie.IECore()
+    net = ie_core.read_network(model=test_net_xml, weights=test_net_bin)
+    exec_net = ie_core.load_network(net, device)
```

Comment:
I would suggest to explicitly specify "GNA_HW" mode here to make sure that subsequent set_config overrides it. And add assert before set_config, too:
`assert parameter == "GNA_HW"`